L1Trigger/L1TMuonEndCap/src/EndCapParamsHelper.cc

```diff
@@ -16,11 +16,11 @@ EndCapParamsHelper *  EndCapParamsHelper::readAndWriteFromEventSetup(const L1TMu
 }
 
 EndCapParamsHelper::EndCapParamsHelper(L1TMuonEndCapParams * w) {
```

Comment:
No changes to this file except spaces ... will probably make the review easier if we just leave it untouched.